src/Components/Footer/Footer.jsx

```diff
@@ -1,8 +1,142 @@
 import React from "react";
-import "./footer.scss";
+import "./Footer.scss";
+// npm install react-icons --save
+import {FiSend} from "react-icon/fi";
+// import {MdOutlineTravelExplore} from 'react-icon/md'
+import {AiOutlineTwitter} from 'react-icon/ai'
+import {AiFillYoutube} from 'react-icon/ai'
+import {AiFillInstagram} from 'react-icon/ai'
+import {FaTripafdvisor} from 'react-icon/Fa'
+import {FiChevronRight} from 'react-icon/Fi'
 
```

Comment:
the spelling problem in importing
import { AiOutlineTwitter } from "react-icons/ai";
import { AiFillYoutube } from "react-icons/ai";
import { AiFillInstagram } from "react-icons/ai";
import { FaTripadvisor } from "react-icons/fa";
import { FiChevronRight } from "react-icons/fi";